cpp/perspective/src/cpp/emscripten.cpp

```diff
@@ -1064,7 +1064,8 @@ namespace binding {
             
             // Always use the `Table` column names and data types on up
             if (table_initialized && is_update) {
-                auto schema = gnode->get_output_schema();
+                auto gnode_output_schema = gnode->get_output_schema();
+                auto schema = gnode_output_schema.drop({"psp_okey"});
```

Comment:
It is amazing to me that this coerces to `std::set<std::string>` ..

Since `column_names` is simply iterated on 10loc below this, it may be cheaper to simple `continue` on `"psp_okey"`, since `drop()` iterates through the set and reconstructs both name and type vectors from scratch.